tests/cases.py

```diff
@@ -351,6 +351,7 @@ def arrow_table_all_data_types(object_format: TArrowFormat, include_json: bool =
         "binary": [random.choice(ascii_lowercase).encode() for _ in range(num_rows)],
         "decimal": [Decimal(str(round(random.uniform(0, 100), 4))) for _ in range(num_rows)],
         "bool": [random.choice([True, False]) for _ in range(num_rows)],
+        "string_null": [random.choice(ascii_lowercase) for _ in range(num_rows - 1)] + [None],
```

Comment:
create a column with only None. and then in panda frame you'll get a column without a type and our schema conversion routine fails

please try to reproduce. if you cannot I'll do that (ConnectorX is doing that when you select a column that has only None)

Reply:
Ok, yeah I see, it comes up as `pyarrow.null()` type. How would be best to handle this? Could just remove such columns before writing, think that makes sense.

Reply:
you can create column without data type as well. it is called "incomplete" and will not be created in the destination

Reply:
Tried this, but duckdb complains when there are extra columns in the parquet. So think we have to delete them.